src/sql/_testing.py

```diff
@@ -187,6 +188,9 @@ def database_ready(
             eng.close()
             print(f"{database} is initialized successfully")
             return True
+        except ModuleNotFoundError as e:
+            msg = "Please install " + e.name.lower() + " first!"
+            pytest.exit(msg)
```

Comment:
We should raise an error (i.e. `raise BaseException(msg)`) instead of stopping the test using `pytest.exit(msg)`.

This way, the error is more informative and the entire unit test actually fails.

pytest.exit(msg)
![image (1)](https://github.com/ploomber/jupysql/assets/103026689/7ea4f60c-a392-4e1a-8b9f-ba75ab213879)


raise BaseException(msg)
![image](https://github.com/ploomber/jupysql/assets/103026689/29b4b70d-1d45-4579-8edd-6d02042184f3)